### ISSUE TITLE
Efficiency Ratio

### ISSUE THREAD

**Issue by jovuribe** [QuantConnect/Lean#6044]
Efficiency Ratio implementation
https://tradingtact.com/kaufman-efficiency-ratio/

#### Checklist

- [X] I have completely filled out this template
- [X] I have confirmed that this issue exists on the current `master` branch
- [X] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)

**Comment by AlexCatarino** [comment]
It looks like the Kaufman Efficiency Ratio is already calculated in the [KaufmanAdaptiveMovingAverage.cs#L119](https://github.com/QuantConnect/Lean/blob/master/Indicators/).

**Comment by jovuribe** [comment]
@AlexCatarino Yep, this indicator contains much of the same code from there but not returns the efficiency ratio.

**Comment by ACasteZZ** [comment]
Hey guys! thanks for the awesome work!
I will need your experience to answer a simple question that will totally show how much of a noob i am (started studying qc and python 2 weeks ago)
...I got the Efficiency ratio code (big fan)... how do i get it in my quantConnect so i can call it like an indicator?

Thanks :)

**Comment by jovuribe** [comment]
@ACasteZZ Once the code is merged into Lean, you can call it using self.KER(symbol, period, Resolution). You can find how to use indicators in the documentation.

**Comment by ACasteZZ** [comment]
@jovuribe Thanks. I guess i'll have to keep monitoring this issue to know when that will happen? :)
Thanks for the work! and great timing too
